.travis.yml

```diff
@@ -29,6 +29,7 @@ before_install:
 
     - cp tests/Application/.env.test.dist tests/Application/.env.test
     - set -a && source tests/Application/.env.test && set +a
+    - composer config -g github-oauth.github.com "$TOKEN"
```

Comment:
Why this?

Reply:
This will be removed once I'm done running tests in my fork